app/build.gradle

```
@@ -36,6 +43,24 @@ dependencies {
 
     // Spring Web
     implementation 'org.springframework.boot:spring-boot-starter-web'
+
+/*  Spring Boot Validation
+    - Spring 2.3 부터는 spring-boot-starter-web 의존성 내부에 있던 validation이 사라짐
```

```suggestion
    - Spring Boot 2.3 부터는 spring-boot-starter-web 의존성 내부에 있던 validation이 사라짐
```

Spring Boot를 말씀하시고 싶으셨던 것 같습니다

---

아 맞습니다!
Spring이랑 Spring Boot의 버전 차이가 많이 다른데, 혼동되지 않게 주의해야겠습니다.
감사합니다 :)